PULL_REQUEST.md

```
@@ -0,0 +1,108 @@
+- Tipagens: Zod + TypeScript;
+- Requisições/Integração: Axios;
+
+### Rodando a aplicação
+
+#### Rodando - clonagem + configuração das variáveis de ambiente
+
+Efetue a clonagem dos dois repositórios, front-end e back-end em uma única pasta, de modo a manter a estrutura mais limpa.
+
+Na pasta do back-end, crie um arquivo `.env` com as variáveis de ambiente necessárias (veja o arquivo `.env.example` para referência).
+
+Para o app rodar corretamente, é necessário gerar duas chaves de criptografia para o AES-256 e inseri-las como as variáveis `CRYPTO_SECRET` e `CRYPTO_IV`. [Neste link](https://acte.ltd/utils/randomkeygen) é possível gerá-las de modo randômico. Após gerá-las, coloque a "Basic 16" na variável `CRYPTO_IV` e a "Encryption key 256" na variável `CRYPTO_SECRET`.
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)